### DISCUSSION TITLE
Deployment error on JSON_BUNDLE_PROCESS_DIR

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17730] [Q&A]
Hi Everyone,

I deployed dagster on an ec2 instance, not i can get the ui, but i keep getting this error, can anyone help me

![Screenshot 2023-11-06 at 10 27 49](https://github.com/dagster-io/dagster/assets/2823852/dce5956d-b643-4c4d-95f6-a41f87a29c15)

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699263016297809?thread_ts=1699263016.297809&cid=C01U954MEER).*

**Comment by jmsanders** [comment]
It looks like the error here lives in your user code, not in Dagster's library code. An op is trying to read from an environment variable called `JSON_BUNDLE_PROCESS_DIR` and that environment variable either isn't set or isn't set to a valid file path.
